middleware/error-handler.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const debug = require('debug')('middleware:errorhandler');
+
+// eslint-disable-next-line no-unused-vars
+module.exports = (error, req, res, next) => {
+  debug('Handling %o', error);
+  if (error.code === 'ECONNREFUSED') {
+    error.code = 'ChromeConnectionRefused';
+    error.message = 'Ensure Chrome Headless is running';
+  }
+  if (error.code) {
+    res.status(400);
+  } else {
+    res.status(500);
+  }
+  res.send(error);
```

Comment:
It's not a question of not being useful, so much as not necessarily being meaningful to the client. How does it actually manifest to a client in an error case? Is is plain text, or json, or something else? Does this change depending on whether the error is an instance of `Error` or a raw js object (admittedly I suggested standardising this in another comment)?

My assumption is that called with an `Error` this will give a plain text response of the error message and stack. I'm not sure that is necessarily the ideal scenario, and perhaps a json response would be preferable.